src/middleware/sagas.js

```diff
@@ -28,7 +30,7 @@ export function* doRequestGetCloudProviderAccess() {
 
 export function* doRequestGetCloudProviderKeys(accessToken, authorization) {
   return yield call(
-    doRequest, 'http://localhost:3100/api/v1/cloud/keys',
+    doRequest, ((process.env.NODE_ENV == 'development')? hosts.DEVELOPMENT: (process.env.NODE_ENV == 'production')? hosts.PRODUCTION : '') +  '/api/v1/cloud/keys',
```

Comment:
Instead of joining strings use URL from Javascript https://developer.mozilla.org/en-US/docs/Web/API/URL/URL